src/App/Components/Common/NavProfile/navbar.js

```diff
@@ -1,9 +1,24 @@
+/* eslint-disable jsx-a11y/mouse-events-have-key-events */
```

Comment:
I this should be a class component as it is, and should have the state. then all subcomponents should be stateless components

Reply:
Okay @alainburindi 
Let's sync and see together what you mean